stable/jenkins/README.md

```diff
@@ -189,6 +189,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `serviceAccountAgent.name`        | name of the agent ServiceAccount to be used by access-controlled resources | autogenerated |
 | `serviceAccountAgent.create`      | Configures if an agent ServiceAccount with this name should be created | `false`         |
 | `serviceAccountAgent.annotations` | Configures annotation for the agent ServiceAccount | `{}`                              |
+| `podTemplates`                    | Configures extra pod templates for the default kubernetes cloud | `{}`                              |
```

Comment:
What about putting this under master.JCasC.podTemplates so all the JCasC config values would be grouped together. Another option could be to put it in the agent section as podTemplates.

Reply:
I also think is a nice change. I left a comment in your PR. I think maybe your PR is the one to go. Let's see how we manage to get this feature working the best possible way. I'm on hold (on committing) until we find common ground :smile:

Reply:
Change done to `agent.podTemplates` as we talked in your PR.